### ISSUE TITLE
IM Alg 2 dictionary

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1393]

**Comment by flannery-denny** [comment]
@schanzer I've added the im algebra 2 dictionary in [this commit ](https://github.com/bootstrapworld/curriculum/commit/59f259e753959f323ef57b2e63747d44032e4bfa)

IM's alg2 curriculum:

- discusses sequences throughout (our alg2 materials don't mention sequences at all)
- spends a lot of time on recursive definitions
- has one lesson focused on populating spreadsheets
- does not include real world datasets or periodic functions
- focuses on concrete examples that students can visualize such as:
-- Sierpinski Triangles
-- Paper cutting
-- Tower of Hanoi

**Comment by flannery-denny** [comment]
@schanzer I've added the im algebra 2 dictionary in these 2 commits:
- https://github.com/bootstrapworld/curriculum/commit/59f259e753959f323ef57b2e63747d44032e4bfa
- https://github.com/bootstrapworld/curriculum/commit/d1e7f4fa48ee73c8dadc7922cd3dad0ade635d00

Most relevant portions of their Alg2 curriculum for our work:
- [Unit 7 Statistical Inferences](https://curriculum.illustrativemathematics.org/HS/students/3/7/index.html) - some overlap with our other DS units
<img width="655" alt="Screen Shot 2023-06-15 at 11 20 37 AM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/e031730c-3e3d-4af4-a7c6-4660bee5c04a">

- [Unit 4 Exponential (and logarithmic) Functions and Equations](https://curriculum.illustrativemathematics.org/HS/students/3/4/index.html)
- [Lesson 5.11 Making a Model for Data](https://curriculum.illustrativemathematics.org/HS/students/3/5/11/index.html)

Their algebra 2 course starts with a unit that:
- discusses sequences throughout (our alg2 materials don't mention sequences at all)
- spends a lot of time on recursive definitions
- has one lesson focused on populating spreadsheets
- focuses on concrete examples that students can visualize such as:
-- Sierpinski Triangles
-- Paper cutting
-- Tower of Hanoi

**Comment by flannery-denny** [comment]
@schanzer @shriram 

The most relevant portions of [im algebra 2](https://curriculum.illustrativemathematics.org/HS/students/3/index.html) curriculum for our work seem to be:
- [Unit 7 Statistical Inferences](https://curriculum.illustrativemathematics.org/HS/students/3/7/index.html) - some overlap with our other DS units
<img width="655" alt="Screen Shot 2023-06-15 at 11 20 37 AM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/e031730c-3e3d-4af4-a7c6-4660bee5c04a">

- [Unit 4 Exponential (and logarithmic) Functions and Equations](https://curriculum.illustrativemathematics.org/HS/students/3/4/index.html)
- [Lesson 5.11 Making a Model for Data](https://curriculum.illustrativemathematics.org/HS/students/3/5/11/index.html)

Their algebra 2 course starts with a unit that:
- discusses sequences throughout (our alg2 materials don't mention sequences at all)
- spends a lot of time on recursive definitions
- has one lesson focused on populating spreadsheets
- focuses on concrete examples that students can visualize such as:
-- Sierpinski Triangles
-- Paper cutting
-- Tower of Hanoi

I've added the lessons as a dictionary in these 2 commits 
- https://github.com/bootstrapworld/curriculum/commit/d1e7f4fa48ee73c8dadc7922cd3dad0ade635d00
- https://github.com/bootstrapworld/curriculum/commit/59f259e753959f323ef57b2e63747d44032e4bfa

And started a list of data referenced and other noticings in #1392

**Comment by shriram** [comment]
I'm curious, what about the earlier parts? Why wouldn't we try to integrate our work into the preceding modules, like [exponential functions](https://curriculum.illustrativemathematics.org/HS/students/3/4/index.html)?

**Comment by shriram** [comment]
Also, those recursion examples: [ugggh!](https://parentheticallyspeaking.org/articles/how-not-to-teach-recursion/)

**Comment by flannery-denny** [comment]
@shriram thanks for sharing that article on recursion. Noting that I had already listed Unit 4 on exponential functions as a suggested alignment in my comment to you.

**Comment by shriram** [comment]
Hi @flannery-denny. Sorry, not sure what you mean. I'm just commenting on their book. Not that we can change it or use anything from my article to fix it.

**Comment by schanzer** [comment]
@shriram can you say why you wanted this issue reopened?

**Comment by schanzer** [comment]
@shriram aaah, understood. I thought you were talking about your recursion comment. Thanks for clarifying, @flannery-denny , what do you think?

**Comment by shriram** [comment]
@schanzer: my email to you contained the *very same link* that I provided above.

**Comment by schanzer** [comment]
@shriram Sorry - on my phone the github app does open links to specific comments. Only the issue itself. I assumed you meant the latter.

When an issue is closed, GitHub also shows _who_ closed it. This would have allowed you to ask Flannery directly, instead of relying on me as an intermediary.

**Comment by shriram** [comment]
Emmanuel, I'm well aware of how GitHub works. But it is also git protocol to explain why an issue is being closed, if it's not obvious from context. I do not understand why @flannery-denny closed this issue while leaving [this comment](https://github.com/bootstrapworld/curriculum/issues/1393#issuecomment-1593330413) untouched.

**Comment by flannery-denny** [comment]
Sorry @shriram. Haven't worked much with you on git. We have lots of ways of communicating with each other on the curriculum team, including an agenda of conversations to continue offline.

I didn't comment on your comment about chapter 4 because it wasn't a new idea. I had already listed the chapter you suggested in my message to y'all and didn't see utility in telling you that.

That issue was made for the purpose of creating a dictionary, which I completed, and I had meant to close the issue with my original comment to you and @schanzer so was just closing it because I noticed I had accidentally left it open.
